### PR TITLE
Add player and controller labels, rename function

### PR DESCRIPTION
**Pull request by MaciekBaron**
Left the `nop` white-space changes in so that they get taken care of.

### CODE REVIEW

**Comment by MaciekBaron** [comment]
TIL you can hide whitespace changes when reviewing, so hopefully that will make things easier :)

File: src/code_800029B0.c [hunk 1]
```diff
@@ -128,7 +128,7 @@ void func_800029B0(void) {
 }
 
 void func_80002A18(void) {
-    struct Controller *temp_v0;
+    struct Controller *currentController;
```
[the remark this answers is not in the document]
Comment (MaciekBaron) [reply src/code_800029B0.c]: Done

**Comment by MegaMech** [comment]
I'm sorry, I had so many PRs, I wasn't sure the best order to merge them. Since yours was mostly renamings, I thought that would be the easiest to leave to last. So, hopefully it's not too big of an issue to fix the conflicts.

**Comment by MaciekBaron** [comment]
No worries, resolved.

**Review by MegaMech — APPROVED** [review]

**Pull request merged**